hyper/common/exceptions.py

```diff
@@ -71,3 +71,22 @@ class MissingCertFile(Exception):
     The certificate file could not be found.
     """
     pass
+
+
+# Create our own ConnectionError.
+try:  # pragma: no cover
+    ConnectionError = ConnectionError
+except NameError:  # pragma: no cover
+    class ConnectionError(Exception):
+        """
+        An error occurred during connection to a host.
+        """
```

Comment:
Hrm. Here's a question: on Pythons that have `ConnectionError`, probably we shouldn't create `hyper.common.exceptions.ConnectionError` as an *alias* of that exception but as a subclass instead. Mind having this inherit instead?